test/Be.Vlaanderen.Basisregisters.Shaperon.Tests/Geometries/GeometryCustomizations.cs

```diff
@@ -157,6 +208,81 @@ public static void CustomizePolygon(this IFixture fixture)
             );
         }
 
+        public static void CustomizeMultiPolygon(this IFixture fixture)
+        {
+            fixture.Customize<Polygon>(customization =>
+                customization.FromFactory(generator =>
+                {
+                    var polygonCount = generator.Next(1, 4);
+                    var polygons = new NetTopologySuite.Geometries.Polygon[polygonCount];
+                    for (var polygonIndex = 0; polygonIndex < polygonCount; polygonIndex++)
+                    {
+                        var offsetX = 10.0 * polygonIndex;
+                        var offsetY = 10.0 * polygonIndex;
+
+                        var shell = new NetTopologySuite.Geometries.LinearRing(
```

Comment:
The data is only semi randomized - future optimization is to generate points in a certain orientation.